spec/models/service_template_spec.rb

```diff
@@ -63,6 +63,21 @@
     end
   end
 
+  context "#prov_type_display" do
+    before(:each) do
```

Comment:
I realized this is used elsewhere in this file, but you do not need to include `(:each)` as it is the default.